messages/device-added-notification.json

```diff
@@ -38,7 +38,419 @@
+                "description": "Types of the device"
+              }
+            },
+            "id": {
+              "type": "string",
+              "title": "Device ID",
+              "description": "ID of the device"
+            },
+            "title": {
+              "type": "string",
+              "title": "Device title",
+              "description": "Title of the Device"
+            },
+            "description": {
+              "type": "string",
+              "title": "Device Description",
+              "description": "Description of the device"
+            },
+            "properties": {
+              "type": "object",
+              "patternProperties": {
+                ".*": {
+                  "type": "object",
+                  "title": "Property",
+                  "description": "Description of the Property",
+                  "required": [
+                    "type",
+                    "visible"
+                  ],
+                  "properties": {
+                    "@type": {
+                      "type": "string",
+                      "title": "Property type",
+                      "description": "The type of the property"
+                    },
+                    "title": {
+                      "type": "string",
+                      "title": "Property title",
+                      "description": "The title of the property"
+                    },
+                    "description": {
+                      "type": "string",
+                      "title": "Property Description",
+                      "description": "Description of the property"
+                    },
+                    "links": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "title": "Property Link",
+                        "description": "A property link",
+                        "required": [
+                          "href",
+                          "rel"
+                        ],
+                        "properties": {
+                          "href": {
+                            "type": "string",
+                            "title": "Link href",
+                            "description": "The href of the link"
+                          },
+                          "rel": {
+                            "type": "string",
+                            "title": "Link rel",
+                            "description": "The type of the relationship"
+                          },
+                          "mediaType": {
+                            "type": "string",
+                            "title": "Link media type",
+                            "description": "The media type of the link"
+                          }
+                        }
+                      }
+                    },
+                    "type": {
+                      "type": "string",
```

Comment:
This can actually be limited to the same enum as the action type below.

Reply:
Done